### ISSUE TITLE
Service Connection docs missing guidance on Service Principal ID, et al.

### ISSUE THREAD

**Issue by riverar** [microsoft/windows-dev-center-vsts-extension#76]
I'm attempting to configure a new Windows Dev Center [sic] service connection in Azure Dev Ops and it's not clear what Service Principal ID represents.

The README has outdated graphics/guidance on this 
https://github.com/microsoft/windows-dev-center-vsts-extension#configuring-your-credentials

Screenshot:
![image](https://user-images.githubusercontent.com/475132/67130266-999ba600-f1b5-11e9-9f7a-959eed27faed.png)

**Comment by louis-wuitton** [comment]
Hi, you leave the service principal id and tenant id to be blank. For the Azure tenant ID you can use the azure tenant id and client id from the Azure App registration you use to authenticate.

**Comment by riverar** [comment]
That is incorrect. There is no way to dismiss this dialog and configure this correctly.

**Comment by louis-wuitton** [comment]
Hi folks, please see this comment. https://github.com/microsoft/windows-dev-center-vsts-extension/issues/77#issuecomment-553552395
